### PR TITLE
Update Curiosity dialogs DatasetCard

### PR DESCRIPTION
**Pull request by vineeths96**
Update Curiosity dialogs DatasetCard

There are some entries in the data fields section yet to be filled. There is little information regarding those fields.

### CODE REVIEW

File: datasets/curiosity_dialogs/README.md [hunk 1]
```diff
@@ -46,37 +47,172 @@ task_ids:
...
+* `messages`: List of dialogs between the user and the assistant and their associated attributes
+  * `dialog_acts`: List of actions performed in the dialogs
+  * `facts`:  List of facts returned by the assistant
+    * `fid`: Fact ID 
+    * `source`: Source for the fact
+    * `used`:  Whether facts were used before in the same dialog
+  * `liked`: List of values indicating whether each dialog was liked
+  * `message`: List of dialogs (messages) between the user and the assistant
+  * `message_id`: Message ID
+  * `sender`:
```
Comment (SBrandeis) [inline datasets/curiosity_dialogs/README.md:187]:
```suggestion
  * `sender`:  Message author ID (0 or 1)
```

**Review by SBrandeis — APPROVED** [review]
Thanks a lot for the contribution! Dataset card is much more complete now.

File: datasets/curiosity_dialogs/README.md [hunk 2]
```diff
@@ -46,37 +47,172 @@ task_ids:
...
+* `shuffle_facts`: Whether facts were shuffled
+* `related_entities` : List of fifteen related entities to the focus entity
+* `tag`: Conversation tag
+* `user_id`: User ID
+* `assistant_id`: Assistant ID
+* `is_annotated`:  
+* `user_dialog_rating`:  
+* `user_other_agent_rating`: 
+* `assistant_dialog_rating`: 
+* `assistant_other_agent_rating`:
```
Comment (lhoestq) [inline datasets/curiosity_dialogs/README.md:205]: can you add the feature type for the fields with no information ? And maybe the value range ? (probably 1-5 for the ratings, and 0 or 1 for `is_annotated `)

Comment (vineeths96) [reply datasets/curiosity_dialogs/README.md:205]: Sure. Do I need to put something like `More Information Needed` or `Incomplete Information` to mention that the descriptions are incomplete?

Comment (lhoestq) [reply datasets/curiosity_dialogs/README.md:205]: Yes feel free to also mention explicitly that information is missing for those fields

**Review by lhoestq — APPROVED** [review]
Thank you :)

**Pull request merged**